inspectit-oce-core/src/main/java/rocks/inspectit/oce/core/exporter/PrometheusExporterService.java

```diff
@@ -0,0 +1,59 @@
+package rocks.inspectit.oce.core.exporter;
+
+import io.opencensus.exporter.stats.prometheus.PrometheusStatsCollector;
+import io.opencensus.exporter.stats.prometheus.PrometheusStatsConfiguration;
+import io.prometheus.client.exporter.HTTPServer;
+import lombok.extern.slf4j.Slf4j;
+import lombok.val;
+import org.springframework.stereotype.Component;
+import rocks.inspectit.oce.core.config.model.InspectitConfig;
+import rocks.inspectit.oce.core.service.DynamicallyActivatableService;
+
+import static io.prometheus.client.CollectorRegistry.defaultRegistry;
+
+/**
+ * Service for the Prometheus OpenCensus exporter.
+ * Can be dynamically started and stopped using the exporters.metrics.prometheus.enabled configuration.
+ */
+@Component
+@Slf4j
+public class PrometheusExporterService extends DynamicallyActivatableService {
```

Comment:
I think we should have a Spring integration test for this class.. So something where you have first disabled, then you update the config, check it's enabled and then disable again.. Should not be too hard, but we would actually test multiple things.. What do you think?